Assets/MRTK/Core/Providers/Hands/ArticulatedHandDefinition.cs

```diff
@@ -72,6 +72,27 @@ public bool IsInPointingPose
             }
         }
 
+        /// <summary>
+        /// Calculates whether the current the current joint pose is selecting (air tap gesture).
+        /// </summary>
+        public bool IsPinching
+        {
+            get
+            {
+                MixedRealityPose thumbTip;
+                MixedRealityPose indexTip;
+                if (unityJointPoses.TryGetValue(TrackedHandJoint.ThumbTip, out thumbTip) && unityJointPoses.TryGetValue(TrackedHandJoint.IndexTip, out indexTip))
+                {
+                    if (Vector3.Distance(thumbTip.Position, indexTip.Position) < 0.02f)
```

Comment:
Ah, might be good to move this magic number into a variable, to make the code more readable and make it easier to change in case you use the magic number elsewhere.

Also, I think instead of having a single threshold, this should use two different threshold: an enter threshold and an exit threshold. This is how [leap motion for hololens](https://github.com/HoloLabInc/LeapMotionInputForMRTK/blob/master/Assets/LeapMotionInputSimulation/Providers/LeapMotionHand.cs#L81) contributed by Tarokosu was implemented. I think this will help with accidental release when dragging objects. In testing this myself with my VR headset + Leap Motion, I found that I dropped things too easily when I was trying to drag objects around.

Reply:
Thanks Julia!
Added an additional threshold for determining the exit distance for IsPinching.